### DISCUSSION TITLE
Request for Reserved Slug Names Functionality in laravel-sluggable

### DISCUSSION THREAD

**Discussion by fabiomlferreira** [spatie/laravel-sluggable#260] [Ideas]
Hey there!

I've been using your Spatie laravel-sluggable package and I gotta say, it's pretty neat! Really helps us out on our Laravel projects.

So here's an idea I had - what if we could have a list of reserved slug names? You know, names that are off-limits when we're auto-generating slugs. It could save us from awkward moments where a generated slug messes with the routing or ends up matching a reserved path, like 'admin', 'dashboard', 'login', you get the gist.

I know we can handle this ourselves with Laravel's validation and all, but wouldn't it be cool if the package already had this built in? It could make things way easier for us users.

**Comment by rettigd** [comment]
Other words might include create, delete (ie default routes). **create** is a real problem since /[model]/create would be unreachable by default. I'd prefer not throwing a bunch of reserved words validation errors.
